File: java/src/main/java/co/edu/sena/capitulo6/ejercicios/Ejercicio07.java
package co.edu.sena.capitulo6.ejercicios;

import java.util.Scanner;

public class Ejercicio07 {

	public static void main(String[] args) {
		Scanner teclado = new Scanner(System.in);
		int n, i;
		double p = 1, x, f = 1, s = 1;
		System.out.println("Ingrese valor de x : ");
		x = teclado.nextInt();
		System.out.println("Ingrese valor de n : ");
		n = teclado.nextInt();
		for (i = 1; i <= n - 1; i++) {
			f = f + 1;
			p = p * x;
			s = s * p / f;
		}
		System.out.println("");
		System.out.println("La sumatoria es  : " + s);
	}

}

File: typescript/src/ejercicios/ejercicio07.ts
(function () {
    let  n , i: number ;
    let p  , x , f  , s  : number;  
    x = parseInt(prompt("Ingrese valor de x : "));
    n = parseInt(prompt("Ingrese valor de n : "));
    p = 1 ;
    f = 1 ;
    s = 1;
    for (i = 1; i <= n - 1 ; i ++) {
         f = f + 1;
         p = p * x;
         s = s * p/f;
    }
   
    console.log("La sumatoria es  : " + s);    
})();